src/UI/Core/ComboBox.cpp

```diff
@@ -115,7 +115,8 @@ void ComboBox::onMouseDown(EventHandler::MouseButton button, int x, int y)
 }
 
 
-void ComboBox::onMouseWheel(int x, int y)
+//TODO: Implement in terms of internal ListBox.
```

Comment:
This randomly added `TODO` seems out of place here. Seems like it should become a GitHub issue, or become a new commit (probably in a new PR).

Reply:
Curious, these should have been removed. I must have missed them during one of the many rebases.

Reply:
Fixed

Reply:
The combobox already reacts to the scroll wheel.